vm.go

```diff
@@ -351,6 +353,54 @@ func (vm *VirtualMachineSpec) ExecuteCommand(command string) error {
 	return nil
 }
 
+// RunCommand executes a command on the Virtual Machine
+func (vm *VirtualMachineSpec) RunCommand(command []string) (output string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Minute)
+	defer cancel()
+	system, err := hcs.OpenComputeSystem(ctx, vm.ID)
+	if err != nil {
+		return "", err
+	}
+	defer system.Close()
+
+	if system.OS() != "linux" {
+		return "", ErrNotSupported
```

Comment:
>ErrNotSupported [](http://example.com/codeflow?start=13&length=15)

why only support linux?

Reply:
ok. will add support for windows as well...

Reply:
add support for windows as well. but couldnt test it... i dont have a windows vhdx, if you have one, please share along with any needed instructions. thanks!

Reply:
aksiot has a windows vhdx, [https://dev.azure.com/mariner-org/mariner/_build/results?buildId=222025&view=artifacts&pathAsName=false&type=publishedArtifacts](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=222025&view=artifacts&pathAsName=false&type=publishedArtifacts)

it's a compressed file, copy all aksiotwindows* to a folder and run aksiotwindows-v1.exe

Reply:
BTW, not necessary to do it in this PR.